nss-day-cohort-52/giffygram-scorpion-pilots
# Kll initial branch

#### Changes Made
1. Added fetchPosts & getPosts to data/provider.js 
2. Changed GiffyGram.js to get and display posts
3. Added fetchPosts to Scripts/main.js:18-19
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout kll-initialBranch
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > When the person provides valid login credentials, the person should see all posts from the GiffyGram database
    > Confirm all posts are displaying properly
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
